pkg/runtime/runtime.go

```diff
@@ -999,6 +999,15 @@ func (a *DaprRuntime) Publish(req *pubsub.PublishRequest) error {
 	return a.pubSubs[req.PubsubName].Publish(req)
 }
 
+// PubSubFeatures is an adapter method for the runtime to list features of a PubSub.
+func (a *DaprRuntime) PubSubFeatures(pubsubName string) ([]pubsub.Feature, error) {
```

Comment:
As mentioned above, would separate the pub sub existence check into it's own method.

Reply:
OK.